### DISCUSSION TITLE
Running WRF with RAP data

### DISCUSSION THREAD

**Discussion by goes1617** [NOAA-GSL/GSL-Regional-Model-Forum#26] [Q&A] [answered]
Hello,

I'm trying to run WRF with RAP data which I downloaded from Google cloud, and I'm having some problems.
**1. number of grid in namelist.wps and namelist.input don't agree.** 
I got namelist.wps from here (https://rapidrefresh.noaa.gov/rap_namelist.wps.txt) and namelist.input from the RAP website (https://rapidrefresh.noaa.gov/wrf.nl.txt) and their e_we and e_sn don't match. Which one should I use? 
**2. num_metgrid_soil_levels is 2.** 
I've downloaded both pressure level and native level data, but from the previous discussion in the forum, it seems like pressure level data don't contain soil information, so I'm trying to run with native level data.
According to "num_metgrid_soil_levels" in the namelist.input on the website, it seems like soil level in the metgrid data should be 4, but I'm only getting 2 if I use Vtable and METGRID.TBL in the website. Can I just continue with num_metgrid_soil_levels of 2?
I guess my main question combined from 1 and 2 is.. _where can I get correct versions of namelist.wps, namelist.input, GEOGRID.TBL, Vtable, and METGRID.TBL to run WRF with the current version of RAP data (either native level or pressure level data)?_
**3. SST data archive**
It seems like I'd need SST data to run RAP, and there's only current SST data in the website.
I was wondering if there's any SST data archive publicly available or I could just run without SST data?

Thank you in advance!

**Comment by EricJames-NOAA** [comment]
Hi...sorry for the very late response.  It appears our namelists on the website are out of date...the current RAP domain is larger than the earlier one.  So you should use the e_we and e_sn that are larger.  

Looks like our METGRID.TBL is also very out of date...we should currently have 9 soil levels in the RAP.   Let me see if I can update the files on the webpage.

Regarding the SST data...I know we have SST data archived on the NOAA HPSS, but if you are outside of NOAA, I'm not sure of a publicly available archive.

**Comment by EricJames-NOAA** [comment, marked as answer]
OK, I finally had time to update the webpage.  Your namelist.wps configuration will depend on what type of grid configuration you want to set up for your configuration. Running off the RAP, you should use this METGRID.TBL:

https://rapidrefresh.noaa.gov/METGRID.TBL.RRgsd

And this Vtable:

https://rapidrefresh.noaa.gov/Vtable.RRgsd

If you are wanting to run with RAP-like or HRRR-like settings, you can use the RAP or HRRR WRF namelists:

https://rapidrefresh.noaa.gov/wrf.nl.txt
https://rapidrefresh.noaa.gov/hrrr/wrf.nl.txt

Let us know if you still have problems.

Thanks,
Eric

**Reply by goes1617** [reply]
Thank you so much!
